docs/concepts/type_adapter.md

```diff
@@ -0,0 +1,77 @@
+You may have types that are not `BaseModel`s that you want to validate data against.
+Or you may want to validate a `List[SomeModel]`, or dump it to JSON.
+
+??? api "API Documentation"
+    [`pydantic.type_adapter.TypeAdapter`][pydantic.type_adapter.TypeAdapter]<br>
+
+For use cases like this, Pydantic provides [`TypeAdapter`][pydantic.type_adapter.TypeAdapter],
+which can be used for type validation, serialization, and JSON schema generation without creating a
+[`BaseModel`][pydantic.main.BaseModel].
+
+A [`TypeAdapter`][pydantic.type_adapter.TypeAdapter] instance exposes some of the functionality from
+[`BaseModel`][pydantic.main.BaseModel] instance methods for types that do not have such methods
+(such as dataclasses, primitive types, and more):
+
+```py
+from typing import List
+
+from typing_extensions import TypedDict
+
+from pydantic import TypeAdapter, ValidationError
+
+class User(TypedDict):
+    name: str
+    id: int
+
+UserListValidator = TypeAdapter(List[User])
```

Comment:
```suggestion
UserListAdapter = TypeAdapter(List[User])
```
(and below).

I think better to use `Adapter` than `Validator` to help people realize it's able to do more than validation, also I just think it's a better naming convention and would want to push people toward using it as they read the docs, even if they are only doing validation with the value.

Reply:
Agreed, thanks for the tip.